rest/admin_api.go

```diff
@@ -45,10 +45,17 @@ func (h *handler) handleCreateDB() error {
 	if err != nil {
 		return err
 	}
-	config.Name = dbName
 
 	validateOIDC := !h.getBoolQuery(paramDisableOIDCValidation)
 
+	if config.Name == "" {
```

Comment:
This if statement is not needed as the config name can be set after the validation is done after line 55-57